.github/workflows/prettier.yml

```diff
@@ -0,0 +1,55 @@
+name: prettier
+on:
+  # for pull requests from this repo
+  pull_request:
+  # for pull requests from forks
+  pull_request_target:
+  # for any pushes that may sidestep pr workflow
+  push:
+    branches:
+    - main
+    - master
+    - dev/*
+    - hotfix/*
+jobs:
+  prettier:
+    # filter out the event that shouldn't apply
+    # prevents running twice when not necessary
+    # if push, make sure it's this repo
+    # if pr, make sure it's this repo
+    # if pr_target, make sure it's a fork
+    if: >-
+      (
+        github.event_name == 'push' &&
+        github.event.pull_request.head.repo.full_name == github.repository
+      ) || (
+        github.event_name == 'pull_request' &&
+        github.event.pull_request.head.repo.full_name == github.repository
+      ) || (
+        github.event_name == 'pull_request_target' &&
+        github.event.pull_request.head.repo.full_name != github.repository
+      )
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: ${{ github.head_ref }}
+
+    - name: NPM - Test, Build, Deploy
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12.11.1
```

Comment:
If we update our node version will we remember to update this as well? Pretty sure the answer is 'no', but is it possible to grab this from the package json?

Reply:
I've been pretty good about orchestrating node version changes in the past (usually a global search of the current version). But you're right, a likely situation that we miss this file.  

I can't think of any sustainable way to script changes to node versions that wouldn't just move the problem somewhere else.